backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/model/mapper/AbstractModelMapper.java

```diff
@@ -0,0 +1,26 @@
+package ca.gov.dtsstn.cdcp.api.web.v1.model.mapper;
+
+import java.util.List;
+import org.springframework.hateoas.CollectionModel;
+import org.springframework.hateoas.Links;
+import org.springframework.hateoas.server.core.EmbeddedWrappers;
+import org.springframework.util.Assert;
+public abstract class AbstractModelMapper {
```

Comment:
**Nitpick:** Convention for this project is to separate imports and class declaration with empty line for readability.

```suggestion
import org.springframework.util.Assert;

public abstract class AbstractModelMapper {
```

Reply:
added a separation